src/test/java/stringaddcalculator/InputStringTest.java

```diff
@@ -0,0 +1,39 @@
+package stringaddcalculator;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+class InputStringTest {
+
+    @Test
+    void 기본_구분자로_숫자를_구분한다() {
+        assertThat(new InputString("1,2").getNumbers()).containsExactly(1, 2);
+        assertThat(new InputString("1:2").getNumbers()).containsExactly(1, 2);
+    }
+
+    @Test
+    void 빈문자열_또는_null인_경우에는_0을_반환() {
+        assertThat(new InputString("").getNumbers()).containsExactly(0);
+        assertThat(new InputString(null).getNumbers()).containsExactly(0);
+    }
```

Comment:
검증할 값과 예측되는 값이 동일하네요.
`null` 값은 다음과 같은 방법으로 검증할 수 있어요!

```suggestion
    @ParameterizedTest
    @CsvSource(value = {"null", "''"}, nullValues = {"null"})
    void 빈문자열_또는_null인_경우에는_0을_반환(String input) {
        assertThat(new InputString(input).getNumbers()).containsExactly(0);
    }
```